### ISSUE TITLE
Some properties doesn't work

### ISSUE THREAD

**Issue by kennowski** [JohnCoates/CSSketch#40]
Sketch version 3.7.1 (Latest)
OS El Capitan 10.11.4 (Latest)

When I try to modify margins (don't know about others), doesn't work.

http://cl.ly/0c0i1x0j3K28

**Comment by julienpecorino** [comment]
Yes same in my side, would be awesome to fixe it, this plugin is the FUTURE ;) thanks for this

**Comment by JohnCoates** [comment]
I believe this is related to https://github.com/JohnCoates/CSSketch/issues/42
